### PR TITLE
Add app name to the splash screen

### LINKED ISSUE #1383: Improving the Splash screen
**Actual Behaviour**
Currently, the splash screen is having only the app icon.

**Expected Behaviour**
To improvise the design, the splash screen must have the name of the app or any animation to make it look better

**Steps to reproduce it**
Open the app.

**LogCat for the issue**
N/A

**Screenshots of the issue**

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by yatri1609**
Fixes #1383 

**Changes**: 
Added app name PSLab to the splash screen

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[splash_screen.zip](https://github.com/fossasia/pslab-android/files/2320426/splash_screen.zip)

### CODE REVIEW

**Comment by yatri1609** [comment]
@CloudyPadmal @abhinavraj23 @Avjeet @harsh-2711  Please review this PR

**Comment by CloudyPadmal** [comment]
Yes, let's make the font bigger and bold.

**Comment by yatri1609** [comment]
@CloudyPadmal @abhinavraj23 Please have a look now

File: app/src/main/res/layout/splash_screen.xml [hunk 1]
```diff
@@ -10,7 +10,14 @@
...
         android:layout_width="match_parent"
         android:layout_height="314dp"
-        app:srcCompat="@drawable/logo"
+        android:src="@drawable/logo"
         android:contentDescription="@string/pocket_science_logo" />
 
+    <ImageView
+        android:id="@+id/PSLabText"
+        android:layout_width="271dp"
+        android:layout_height="95dp"
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/splash_screen.xml]: Shall we add these dimensions to resource folder?

**Pull request merged**